### DISCUSSION TITLE
Which Version Is Everyone Using?

### DISCUSSION THREAD

**Discussion by keith8496** [josephdadams/TallyArbiter#563] [Q&A]
Hello everyone!

I discovered this project about four weeks ago and it looks like a great value to the community. I bought three M5StickC Plusses and had them working w/in a few hours. I’ve been modifying the sticks to better suit our environment. I’ve tested several versions of TA (v2.x and v3.x) on Mac and Docker under Linux. 

I know the v3.x pre-releases aren’t officially for production… but GMail was in beta forever… I suspect many of my fellow geeks have found a v3.x release that’s stable enough. So…

**What version are you using in your live environment?** What are your sources? What kind of clients do you have?

**Comment by wrldtvlr** [comment]
The biggest problem with v3 is the plugins that haven't been rewritten.

**Reply by keith8496** [reply]
Are the plugins = listener clients or something else?

**Reply by wrldtvlr** [reply]
Sigh. Disregard. I read Tally Arbiter and was thinking Companion. We've been using v3 of TA for months with no problem.

**Comment by keith8496** [comment]
My modifications to the sticks are based on the v3 master branch. So I’m kinda married to v3 now. 

I currently have v3.0.1 running. I’ve gone back and forth connecting TA directly to our ATEM vs having Bitfocus Companion send OSC to TA. I have tested both. In general we will use Companion to proxy the ATEM’s connection limit and provide automation.

**Comment by josephdadams** [comment]
I use 2.x myself. Found 3.0 to be too unstable with TSL data for some reason and never had time to track it down whereas 2.x does it just fine. One of these days I hope to be able to put more development time into this project but haven’t been able to for a long time because I have needed to use my spare time coding for hire.
